suite-native/module-send/src/components/CoinControl/SendUtxoScreenFooter.tsx

```diff
@@ -0,0 +1,69 @@
+import { useMemo } from 'react';
+import Animated, { SlideInDown } from 'react-native-reanimated';
+
+import { Button, HStack, Text, VStack } from '@suite-native/atoms';
+import { CryptoAmountFormatter } from '@suite-native/formatters';
+import { Translation } from '@suite-native/intl';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+const footerStyle = prepareNativeStyle(() => ({
+    borderTopWidth: 1,
+    borderTopColor: 'rgba(0, 0, 0, 0.1)',
+}));
+
+type CoinControlScreenFooterProps = {
+    selectedTotal: string;
+    onSubmit: () => void;
+    amount?: string;
+};
+
+export const SendUtxoScreenFooter = ({
+    selectedTotal,
+    onSubmit,
+    amount,
+}: CoinControlScreenFooterProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    const missingToAmount = useMemo(
+        () =>
+            amount && amount !== '0' && Number(selectedTotal) < Number(amount)
+                ? Number(amount) - Number(selectedTotal)
+                : null,
+        [amount, selectedTotal],
+    );
+
+    return (
+        <Animated.View entering={SlideInDown}>
+            <VStack style={applyStyle(footerStyle)} padding="sp16" spacing="sp12">
+                <VStack spacing="sp4">
+                    <HStack justifyContent="space-between">
+                        <Text variant="hint">
+                            <Translation id="moduleSend.coinControl.utxos.selected" />
+                        </Text>
+                        <CryptoAmountFormatter
+                            isBalance={false}
+                            variant="hint"
+                            value={selectedTotal}
+                            symbol="btc"
+                        />
+                    </HStack>
+                    {missingToAmount && (
+                        <HStack justifyContent="space-between">
+                            <Text variant="body">
+                                <Translation id="moduleSend.coinControl.utxos.remaining" />
+                            </Text>
+                            <CryptoAmountFormatter
+                                variant="body"
+                                value={missingToAmount}
+                                symbol="btc"
+                                isBalance={false}
+                            />
+                        </HStack>
+                    )}
+                </VStack>
+
+                {!missingToAmount && <Button onPress={onSubmit}>Continue</Button>}
```

Comment:
Hardcoded string value
```suggestion
                {!missingToAmount && <Button onPress={onSubmit}><Translation id="generic.buttons.continue" /></Button>}
```

Reply:
good catch

Reply:
ca79ab1df38447a6c4cfe999a927ae41a793df59